apps/src/p5lab/spritelab/SpriteLab.js

```diff
@@ -89,6 +91,22 @@ export default class SpriteLab extends P5Lab {
     }
   }
 
+  alertStudent(msg) {
+    if (msg) {
```

Comment:
nit: always prefer returning early to reduce nesting (which improves readability) over wrapping all functionality in a conditional

```javascript
if (!msg) {
  return;
}

// now everything else can be at this indentation level
```